assets/i18n/hi.json

```diff
@@ -347,6 +392,27 @@
 	"I18N_SPLASH_THIRD_EXPLORATION_DESCRIPTION": "Oppia के द्वारा कई विषयो पर एक्सप्लोरेशन्स <a href=\"/dashboard?mode=create\">बनाया</a> और <a href=\"/library\">शेयर</a> किये जा सकते है , जो केवल आपकी कप्लना द्वारा सिमित है।",
 	"I18N_SPLASH_TITLE": "पुस्तकों के बाहर सोचे।",
 	"I18N_SUBSCRIBE_BUTTON_TEXT": "सदस्यता लें",
+	"I18N_TEACH_PAGE_COMMUNITY_GUIDELINES_GUIDELINE_1": "<strong> प्रकाशित एक्सप्लोरेशन पर प्रतिक्रिया देने के लिए हर कोई स्वागत है। </strong> हर किसी की मदद से, हम साइट पर लगातार पढ़ सकते हैं और उन्हें यथासंभव प्रभावी बना सकते हैं।",
```

Comment:
Do you know why the corresponding line on en.json is 396? It looks like there might be a translation missing; perhaps that might be worth correcting?